docs/docs/api/rest/README.md

```diff
@@ -640,6 +675,56 @@ curl 'https://api.pipedream.com/v1/orgs/o_abc123/sources' \
 }
 ```
 
+### Get Workspaces's Connected Accounts
+
+---
+
+Retrieve all the connected accounts for a specific workspace.
+
+#### Endpoint
+
+```
+GET /workspaces/<workspace_id>/accounts
+```
+
+#### Path Parameters
+
+`workspace_id` **string**
+
+[Switch to your workspace's context](/workspaces/#switching-between-workspaces) and [find your org's ID](/workspaces/#finding-your-workspace-s-id).
+
+#### Query Parameters
+
+`app_slug` **string** (_optional_)
+
+To look up the connected account information for a specific app, use the `app_slug` query parameter to filter results for that app. You can find a given app's `app_slug` from the [Explore](https://pipedream.com/explore) page by selecting the app, and looking at the prop definition for the app.
```

Comment:
Should this be `query` rather than `app_slug`? It looks like this endpoint uses a `query` parameter.

https://github.com/PipedreamHQ/pipedreamin/blob/783eeff027d868ec905181792675edd83dd6cc04/api/app/controllers/v1/orgs_controller.rb#L30

Reply:
Good catch! Updated to use `query` in all docs and examples for this.